### ISSUE TITLE
TikTok Source Bugged

### ISSUE THREAD

**Issue by Abadima** [DuncteBot/skybot-lavalink-plugin#12]
## Description
TikTok playback doesn't appear to work properly, whenever a tiktok URL (tiktok.com / vm.tiktok.com) is input, it loads then ends instantly, have tested with various tiktok links.

## Version info
**Client used:** erela.js

Output of `java -version`:
```
openjdk version "17.0.3" 2022-04-19
OpenJDK Runtime Environment (build 17.0.3+7-nixos)
OpenJDK 64-Bit Server VM (build 17.0.3+7-nixos, mixed mode, sharing)
```

 Error in playback of 6988277033168096517

com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
    at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:44) ~[lavaplayer-1.3.99.jar!/:na]
    at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:116) ~[lavaplayer-1.3.99.jar!/:na]
    at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$1(DefaultAudioPlayerManager.java:354) ~[lavaplayer-1.3.99.jar!/:na]
    at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[na:na]
    at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[na:na]
    at java.base/java.lang.Thread.run(Thread.java:833) ~[na:na]
    Suppressed: com.sedmelluq.discord.lavaplayer.tools.exception.EnvironmentInformation: 
  lavaplayer.version: 1.3.99-original
  os.arch: amd64
  os.name: Linux
  os.version: 5.15.0-1021-gcp
  java.vendor: N/A
  java.version: 17.0.3
  java.runtime.version: 17.0.3+7-nixos
  java.vm.version: 17.0.3+7-nixos
Caused by: java.lang.NullPointerException: Cannot invoke "String.length()" because "this.input" is null
    at java.base/java.net.URI$Parser.parse(URI.java:3165) ~[na:na]
    at java.base/java.net.URI.<init>(URI.java:623) ~[na:na]
    at com.dunctebot.sourcemanagers.Mp3Track.loadStream(Mp3Track.java:57) ~[skybot-lavalink-plugin-1.4.0.jar!/:na]
    at com.dunctebot.sourcemanagers.tiktok.TikTokAudioTrack.loadStream(TikTokAudioTrack.java:86) ~[skybot-lavalink-plugin-1.4.0.jar!/:na]
    at com.dunctebot.sourcemanagers.tiktok.TikTokAudioTrack.process(TikTokAudioTrack.java:72) ~[skybot-lavalink-plugin-1.4.0.jar!/:na]
    at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:104) ~[lavaplayer-1.3.99.jar!/:na]
    ... 4 common frames omitted

**Comment by duncte123** [comment]
Could you provide links where this happens?

**Comment by Abadima** [comment]
https://www.tiktok.com/@wishdaniel7/video/7159182023335202074
![image](https://user-images.githubusercontent.com/39747270/199642671-bf9759ef-a97e-45df-9dd3-89ad70f8811c.png)
^ That also happens sometimes
https://www.tiktok.com/@pov_joshi/video/7117998632925711622
https://vm.tiktok.com/ZMFDgoVEC/
https://vm.tiktok.com/ZMFDgTaKn/

**Comment by duncte123** [comment]
Thank you, I'll see what I can do to fix this

**Comment by duncte123** [comment]
Fixed in version 1.4.1
